commands/project/__tests__/deploy.test.ts

```diff
@@ -90,127 +108,132 @@ describe('commands/project/deploy', () => {
+      } as ArgumentsCamelCase<deployCommand.ProjectDeployArgs>;
       projectConfig = {
         name: 'project name from config',
+        srcDir: 'src',
+        platformVersion: '2025',
       };
-      getProjectConfig.mockResolvedValue({ projectConfig });
-      projectNamePrompt.mockResolvedValue({ projectName: 'fooo' });
-      getProjectDetailUrl.mockReturnValue(projectDetailUrl);
+      getProjectConfigSpy.mockResolvedValue({
+        projectConfig,
+        projectDir: 'projectDir',
+      });
+      projectNamePromptSpy.mockResolvedValue({ projectName: 'fooo' });
+      getProjectDetailUrlSpy.mockReturnValue(projectDetailUrl);
       uiLinkSpy.mockImplementation(text => {
         return text;
       });
-      getAccountConfig.mockReturnValue({ accountType });
-      fetchProject.mockResolvedValue({ data: projectDetails });
-      deployProject.mockResolvedValue({ data: deployDetails });
+      getAccountConfigSpy.mockReturnValue({ accountType, env: 'qa' });
+      fetchProjectSpy.mockResolvedValue(
+        mockHubSpotHttpResponse(exampleProject)
+      );
+      deployProjectSpy.mockResolvedValue(
+        mockHubSpotHttpResponse(deployDetails)
+      );
 
       // Spy on process.exit so our tests don't close when it's called
-      processExitSpy = jest.spyOn(process, 'exit').mockImplementation(() => {});
+      // @ts-expect-error Doesn't match the actual signature because then the linter complains about unused variables
+      processExitSpy.mockImplementation(() => {});
     });
 
     it('should load the account config for the correct account id', async () => {
-      await deployCommand.handler(options);
-      expect(getAccountConfig).toHaveBeenCalledTimes(1);
-      expect(getAccountConfig).toHaveBeenCalledWith(options.derivedAccountId);
+      await deployCommand.handler(args);
+      expect(getAccountConfigSpy).toHaveBeenCalledTimes(1);
+      expect(getAccountConfigSpy).toHaveBeenCalledWith(args.derivedAccountId);
     });
 
     it('should track the command usage', async () => {
-      await deployCommand.handler(options);
+      await deployCommand.handler(args);
       expect(trackCommandUsage).toHaveBeenCalledTimes(1);
       expect(trackCommandUsage).toHaveBeenCalledWith(
         'project-deploy',
         { type: accountType },
-        options.derivedAccountId
+        args.derivedAccountId
       );
     });
 
     it('should load the project config', async () => {
-      await deployCommand.handler(options);
-      expect(getProjectConfig).toHaveBeenCalledTimes(1);
-      expect(getProjectConfig).toHaveBeenCalledWith();
-    });
-
-    it('should load the project config', async () => {
-      await deployCommand.handler(options);
-      expect(getProjectConfig).toHaveBeenCalledTimes(1);
-      expect(getProjectConfig).toHaveBeenCalledWith();
+      await deployCommand.handler(args);
+      expect(getProjectConfigSpy).toHaveBeenCalledTimes(1);
+      expect(getProjectConfigSpy).toHaveBeenCalledWith();
```

Comment:
Can this be replaced with `toHaveBeenCalled`, if it doesn't take any arguments?

Reply:
Yep, done in [da4a9ed](https://github.com/HubSpot/hubspot-cli/pull/1408/commits/da4a9edf1299c72980f7ebfffe19ebd5c44c8f6b) 👍